tests/features/solution/solution_overview.feature

```diff
@@ -0,0 +1,101 @@
+      | NonProfit example | Non-Profit Organisation |
+    And the following contact:
+      | email | foo@bar.com         |
+      | name  | Information example |
+    # Check that visiting as an anonymous does not create cache for all users.
+    When I am an anonymous user
+    And I am on the homepage
+    Then I should see the link "Solutions"
+    And I click "Solutions"
+
+    # Check page for authenticated users.
+    When I am logged in as "Madam Shirley"
+    And I am on the homepage
+    And I click "Solutions"
+    Then I should see the text "Non electronic health"
+    And I should see the text "Closed data"
+    And I should see the text "Isolating Europe"
+
+    When I am an anonymous user
+    And I am on the homepage
+    Then I should see the link "Solutions"
+    When I click "Solutions"
+    Then I should see the link "Non electronic health"
+    And I should see the text "Supports health-related fields"
+    And I should see the link "Closed data"
+    And I should see the text "Facilitate access to data sets"
+    And I should see the link "Isolating Europe"
+    And I should see the text "Reusable tools and services"
+    When I click "Non electronic health"
+    Then I should see the heading "Non electronic health"
+
+    # Add new solution as a moderator to directly publish it.
+    Given I am logged in as a moderator
+    When I am on the homepage
+    And I click "Propose solution"
+    Then I should see the heading "Propose solution"
+    When I fill in the following:
+      | Title       | Colonies in Earth                   |
+      | Description | Some space mumbo jumbo description. |
+      | Documentation    | text.pdf                                                               |
+      | Spatial coverage | Belgium (http://publications.europa.eu/resource/authority/country/BEL) |
+      | Language         | http://publications.europa.eu/resource/authority/language/VLS          |
+    Then I select "http://data.europa.eu/eira/TestScenario" from "Solution type"
+    And I select "Demography" from "Policy domain"
+    And I attach the file "text.pdf" to "Documentation"
+    And I attach the file "logo.png" to "Logo"
+    And I attach the file "banner.jpg" to "Banner"
+    # Click the button to select an existing contact information.
+    And I press "Add existing" at the "Contact information" field
+    And I fill in "Contact information" with "Information example"
+    And I press "Add contact information"
+    # Click the button to select an existing owner.
+    And I press "Add existing" at the "Owner" field
+    And I fill in "Owner" with "NonProfit example"
+    And I press "Add owner"
+    And I press "Publish"
+    Then I visit the "Colonies in Earth" solution
+    Then I should see the text "Colonies in Earth"
+
+    And I am on the homepage
+    And I click "Solutions"
+    Then I should see the text "Colonies in Earth"
+
+    # @todo: Normally the solution should go through a moderation process.
```

Comment:
I initially intended to have it as a reminder that this should have been created by a non moderator user and be published by a moderator. But this is not part of this test so we can accept that it is directly published by a moderator.
Removing the comments.